main.go

```diff
@@ -60,6 +60,7 @@ var (
 	gitScanURI          = gitScan.Arg("uri", "Git repository URL. https://, file://, or ssh:// schema expected.").Required().String()
 	gitScanIncludePaths = gitScan.Flag("include-paths", "Path to file with newline separated regexes for files to include in scan.").Short('i').String()
 	gitScanExcludePaths = gitScan.Flag("exclude-paths", "Path to file with newline separated regexes for files to exclude in scan.").Short('x').String()
```

Comment:
Do you think that we should deprecate the exclude path regex files longer term? It seems less economic to me than globbing, which is more prevalent among CLI tools that deal with paths and files.